content/pt-br/docs/concepts/services-networking/network-policies.md

```diff
@@ -0,0 +1,348 @@
+
+Veja o tutorial [Declarando uma política de redes](/docs/tasks/administer-cluster/declare-network-policy/) para mais exemplos.
+
+## Comportamento dos seletores `to` e `from`
+
+Existem quatro tipos de seletores que podem ser especificados nas sessões `ingress.from` ou 
+`egress.to`:
+
+__podSelector__: Seleciona Pods na mesma namespace que a política de rede foi criada, e que deve 
+ser permitido origens no tráfego de entrada ou destinos no tráfego de saída.
+
+__namespaceSelector__: Seleciona namespaces para o qual todos os Pods devem ser permitidos como 
+origens no caso de tráfego de entrada ou destino no tráfego de saída.
+
+__namespaceSelector__ *e* __podSelector__: Uma entrada `to`/`from` única que permite especificar 
+ambos `namespaceSelector` e `podSelector` e seleciona um conjunto de Pods dentro de uma namespace.
+Seja cuidadoso em utilizar a sintaxe YAML correta; essa política:
+
+```yaml
+  ...
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          user: alice
+      podSelector:
+        matchLabels:
+          role: client
+  ...
+```
+contém um único elemento `from` permitindo conexões de Pods com a label `role=client` em 
+namespaces com a _label_  `user=alice`. Mas *essa* política:
+
+```yaml
+  ...
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          user: alice
+    - podSelector:
+        matchLabels:
+          role: client
+  ...
+```
+
+contém dois elementos no conjunto `from` e permite conexões de Pods na namespace local com 
+a _label_ `role=client`, *OU* de qualquer outro Pod em qualquer outra namespace que tenha 
+a label `user=alice`.
+
+Quando estiver em dúvida, utilize o comando `kubectl describe` para verificar como o 
+Kubernetes interpretou a política.
+
+__ipBlock__: Isso seleciona um conjunto particular de faixas de IP a serem permitidos como 
+origens no caso de entrada ou destinos no caso de saída. Devem ser considerados IPs externos
+ao cluster, uma vez que os IPs dos Pods são efêmeros e imprevisíveis.
+
+Os mecanismos de entrada e saída do cluster geralmente requerem que os IPs de origem ou destino
+sejam reescritos. Em casos em que isso aconteça, não é definido se deve acontecer antes ou 
+depois do processamento da `NetworkPolicy` que corresponde a esse tráfego, e o comportamento
+pode ser diferente para cada plugin de rede, provedor de nuvem, implementação de `Service`, etc.
+
+No caso de tráfego de entrada, isso significa que em alguns casos você pode filtrar os pacotes 
+de entrada baseado no IP de origem atual, enquanto que em outros casos o IP de origem que 
+a `NetworkPolicy` atua pode ser o IP de um `LoadBalancer` ou do Nó em que o Pod está executando.
+
+No caso de tráfego de saída, isso significa que conexões de Pods para `Services` que são reescritos
+para IPs externos ao cluster podem ou não estar sujeitos a políticas baseadas no campo `ipBlock`.
+
+For egress, this means that connections from pods to `Service` IPs that get rewritten to
```

Comment:
Essa parte ficou em inglês :)

Reply:
Can you translate and I will approve next! Thanks!

Reply:
Actually I've translated in the above sentence and forgot to remove 😓 

Removed